sidecar/src/figwheel_sidecar/build_middleware/javascript_reloading.clj

```diff
@@ -12,7 +12,9 @@
 (defn cljs-target-file-from-foreign [output-dir file-path]
   (first (filter #(.exists %)
                  ;; try the projected location
-                 [(io/file output-dir (utils/relativize-local file-path))
+                 [(if-not (.isAbsolute (io/file file-path))
```

Comment:
Would it make more sense to use an `if` here rather than `if-not` to avoid the double negative?